cmd/minikube/cmd/start.go

```diff
@@ -1659,6 +1659,13 @@ func validateKubernetesVersion(old *config.ClusterConfig) {
 	newestVersion := semver.MustParse(strings.TrimPrefix(constants.NewestKubernetesVersion, version.VersionPrefix))
 	zeroVersion := semver.MustParse(strings.TrimPrefix(constants.NoKubernetesVersion, version.VersionPrefix))
 
+	paramVersion := viper.GetString(kubernetesVersion)
```

Comment:
Capital V does not work. Do we want it so that the paramVersion is completely set to lowercase so that this is accounted for? Or do we want it to specifically set the first character to lowercase? I think the former would mean someone could input something like "StAble" as the flag and it would recognize it as "stable"

Reply:
yeah lets convert the user input to lowercase for all cases, that makes our life easier